.github/workflows/python-package.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master, development ]
```

Comment:
We will be removing Development branch since we haven't had a official release yet we will continue to use only master at the moment.

Reply:
@shahraizali  @sbalasa 
I will close this PR and open up a new PR based on the master branch. I will also remove the Github actions for now. I will open up a new issue for the Github actions.

Reply:
Here is the issue: https://github.com/thenewboston-developers/thenewboston-python-client/issues/38